federated_learning/financial_fraud_detection/README.md

```diff
@@ -80,6 +88,22 @@ cd tree-based/
 nvflare simulator jobs/creditcard_5_bagging_IID_split_uniform_lr -w ${PWD}/workspaces/xgboost_workspace_5_bagging_exponential_split_scaled_lr -n 5 -t 5
 ```
 
+You will see the execution as shown in the picture.
+![execution](./figures/execution.png)
+
+Each client loads the global model and starts local training. The application will periodically print the train-auc (AUC calculated from the training dataset) and valid-auc (AUC calculated from the validation dataset).
+After the federated learning phase finishes, the server will store the global model.
+![execution](./figures/store_model.png)
+
+#### Step 4. Prediction by the global model
+
+This is a simple example that utilizes a specific piece of data from the dataset to confirm that the prediction result is consistent with the ground truth. Additionally, you have the flexibility to modify the prediction inputs yourself based on your needs.
```

Comment:
I'm not sure the means of "prediction inputs."
Is the last sentence means that users can change the "specific piece of data" mentioned in the first sentence to suit their needs?

Reply:
Yes, as I use the specific piece of data to do this prediction example. The users can modify based on their needs.